### DISCUSSION TITLE
disabilitazione flag enasarco su tutte le fatture tramite api v2

### DISCUSSION THREAD

**Discussion by rastellid** [fattureincloud/api#248] [Q&A] [answered]
Ciao 

Utilizzando le api v2 avrei necessita di disabilitare per qualsiasi fattura il flag Enasarco che mi viene impostato di default a true ogni volta che viene creato un nuovo documento.

Mi sapete dire a qualche chiave devo fare riferimento ?

**Comment by valmoz** [comment, marked as answer]
Buongiorno @rastellid 
ha già provato a seguire questa guida? [Enesarco](https://secure.fattureincloud.it/help/articolo/386-imposta-ritenuta-enasarco)

Grazie
Mauro

**Reply by rastellid** [reply]
Buongiorno @valmoz 

Il passaggio della guida l'ho eseguito, ma quando creo una fattura con le API v2 di default mi imposta la ritenuta Enesarco, di default a 1% 

Invece vorrei che nessuna ritenuta venisse applicata

**Reply by emajo** [reply]
Ciao @rastellid, le consiglio di scrivere al nostro supporto clienti così da poter vedere in dettaglio i log e le impostazioni del suo account.

**Reply by valmoz** [reply]
Buongiorno @rastellid 
è riuscito a risolvere con il nostro supporto clienti?

Grazie
Mauro
